save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/animations/Particles.kt

```diff
@@ -0,0 +1,20 @@
+@file:JsModule("react-tsparticles")
+@file:JsNonModule
+
+
+package com.saveourtool.save.frontend.externals.animations
+
+import react.*
+
+@JsName("Particles")
```

Comment:
```suggestion
@JsName("default")
```
because they use `export default class Particels` and not simply `export class Particles`?